src/views/Affiliates/CriteriaModal.tsx

```diff
@@ -0,0 +1,330 @@
+  affiliateEarnRate: string;
+}
+
+export const CriteriaModal = ({
+  isCriteriaVisible,
+  accountStats,
+  toggleCriteria,
+  stakedAmount,
+  userTier,
+}: {
+  isCriteriaVisible: boolean;
+  toggleCriteria: () => void;
+  accountStats?: IAffiliateStats;
+  stakedAmount?: string;
+  userTier: number | 'vip';
+}) => {
+  const { isConnectedWagmi } = useWalletConnection();
+
+  const stringGetter = useStringGetter();
+  const stakedDYdX = stakedAmount ?? '0';
+
+  const tiers: ITierDefinition[] = [
+    {
+      tier: 0,
+      requirements: { staked: 0, referredVol: 0 },
+      affiliateEarnRate: '5.0%',
+    },
+    {
+      tier: 1,
+      requirements: { staked: 200, referredVol: 1000000 },
+      affiliateEarnRate: '10.0%',
+    },
+    {
+      tier: 2,
+      requirements: { staked: 1000, referredVol: 5000000 },
+      affiliateEarnRate: '12.5%',
+    },
+    {
+      tier: 3,
+      requirements: { staked: 5000, referredVol: 25000000 },
+      affiliateEarnRate: '15.0%',
+    },
+    {
+      tier: 'vip',
+      requirements: {} as { referredVol: number; staked: number },
+      affiliateEarnRate: '50.0%',
+    },
+  ];
+
+  const currentUserTierIdx = tiers.findIndex((tier) => tier.tier === userTier);
+
+  const currentUserTier = tiers[currentUserTierIdx];
+
+  return (
+    <$Dialog
+      isOpen={isCriteriaVisible}
+      setIsOpen={toggleCriteria}
+      placement={DialogPlacement.Default}
+      title={
+        !isConnectedWagmi
+          ? stringGetter({ key: STRING_KEYS.AFFILIATE_TIERS })
+          : userTier === 'vip'
+            ? stringGetter({
+                key: STRING_KEYS.YOURE_A_VIP,
+                params: {
+                  VIP: (
+                    <span className="text-color-success">
+                      {stringGetter({ key: STRING_KEYS.VIP })}
+                    </span>
+                  ),
+                },
+              })
+            : stringGetter({
+                key: STRING_KEYS.YOUR_TIER,
+                params: {
+                  TIER: currentUserTier.tier,
+                },
+              })
+      }
+      hasHeaderBlur={false}
+    >
+      <$Container className="flex flex-col gap-y-1">
+        <div className="flex flex-col gap-y-1 px-1 notTablet:p-0">
+          <div className="text-color-text-0">
+            {userTier === 'vip'
+              ? stringGetter({
+                  key: STRING_KEYS.PROGRAM_CARD_BODY_VIP,
+                  params: {
+                    VIP: stringGetter({ key: STRING_KEYS.VIP }),
+                  },
+                })
+              : stringGetter({
+                  key: STRING_KEYS.CRITERIA_MODAL_VIP_DISCLAIMER,
+                  params: {
+                    VIP_VALUE: <span className="text-color-text-1">{'{VIP Value}'}</span>,
+                    REGULAR_VALUE: <span className="text-color-text-1">{'{Regular Value}'}</span>,
+                    APPLY_HERE: (
+                      <a href="https://dydx-affiliates.fuul.xyz/">
```

Comment:
lets use `<Link isInline href={url}>`

We try to have the URLs come from the config as well:
https://github.com/dydxprotocol/v4-web/blob/49621c4d54639798e443335d5044835e272ef638/public/configs/v1/env.json#L56

and access via useURLConfigs.

There should be examples you can find in the code base!